SCRIPTS.md

```diff
@@ -78,6 +78,14 @@ Keeps the browser open in debug mode so that you can set break points and reload
 npm test -- --packages @webex/webex-core --browser --karma-debug
 ```
 
+# Run the samples automation tests locally
```

Comment:
Docs updates should be in a separate commit: `docs(scripts): add samples automation docs`